README.md

```diff
@@ -103,7 +104,7 @@ newPIN := fmt.Sprintf("%06d", newPINInt)
 newPUK := fmt.Sprintf("%08d", newPUKInt)
 
 // Set all values to a new value.
-if err := yk.SetManagementKey(piv.DefaultManagementKey, newKey); err != nil {
+if err := yk.SetManagementKey(piv.DefaultManagementKey, newKey[:]); err != nil {
```

Comment:
instead of slicing every time, create a slice above instead of a fixed size array

newKey := make([]byte, 24)

Reply:
Good call, I didn't think that one through.